### PR TITLE
Updated ControlledToken to use ERC20PermitUpgradeable

### PR DESCRIPTION
**Pull request by asselstine**

### CODE REVIEW

File: contracts/token/ControlledToken.sol [hunk 1]
```diff
@@ -30,6 +30,7 @@ contract ControlledToken is ERC20Upgradeable, ControlledTokenInterface {
     initializer
   {
     __ERC20_init(_name, _symbol);
+    __ERC20Permit_init("PoolTogether Token");
```
Comment (aodhgan) [inline contracts/token/ControlledToken.sol:33]: Would it be better to change to "PoolTogether ControlledToken" to ensure people are not confused with POOL?

File: test/helpers/signPermit.js [hunk 2]
```diff
@@ -8,18 +9,20 @@ const domainSchema = [
...
 
 async function signPermit(signer, domain, message) {
     let myAddr = await signer.getAddress();
 
-    if (myAddr.toLowerCase() !== message.holder.toLowerCase()) {
-        throw(`signPermit: address of signer does not match holder address in message`);
+    debug(`signPermit(): ${myAddr}`)
+
+    if (myAddr.toLowerCase() !== message.owner.toLowerCase()) {
+        throw(`signDaiPermit: address of signer does not match holder address in message`);
```
Comment (aodhgan) [inline test/helpers/signPermit.js:25]: nit: signDaiPermit -> signPermit?

File: contracts/token/ControlledToken.sol (hunk 1 above)
Comment (asselstine) [reply contracts/token/ControlledToken.sol:33]: Yeah- I like that

**Comment by aodhgan** [comment]
nits aside, LGTM

**Pull request merged**